cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,33 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        db_conn = None
+        max_attempts = 5
+        attempts = 0
```

Comment:
As a suggestion, you might want to consider moving the `max_attempts` variable to a configuration file or environment variable. This would make it easier to adjust the number of attempts without having to modify the code directly.